src/coins.ts

```diff
@@ -847,18 +847,16 @@ const basicCoins: BasicCoin[] = [
         decimals: 6,
       },
       [ChainId.ARB]: {
-        address: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
+        address: '0xaf88d065e77c8cC2239327C5EDb3A432268e5831',
         decimals: 6,
       },
       [ChainId.ONE]: {
         address: '0x985458e523db3d53125813ed68c274899e9dfab4',
         decimals: 6,
       },
       [ChainId.AVA]: {
-        address: '0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664',
+        address: '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e',
         decimals: 6,
-        symbol: 'USDC.e',
-        name: 'USD Coin',
```

Comment:
Currently the symbol and name of all the USDC coins will be `USDC`. Should we instead set the name to `USD Coin` for all USDC token?

Reply:
Yep